ReactiveCocoa/Swift/FoundationExtensions.swift

```diff
@@ -9,13 +9,13 @@
 import Foundation
 import enum Result.NoError
 
-extension NSNotificationCenter {
+extension NotificationCenter {
 	/// Returns a producer of notifications posted that match the given criteria.
 	/// If the `object` is deallocated before starting the producer, it will 
 	/// terminate immediatelly with an Interrupted event. Otherwise, the producer
 	/// will not terminate naturally, so it must be explicitly disposed to avoid
 	/// leaks.
-	public func rac_notifications(name: String? = nil, object: AnyObject? = nil) -> SignalProducer<NSNotification, NoError> {
+	public func rac_notifications(for name: Notification.Name, object: AnyObject? = nil) -> SignalProducer<Notification, NoError> {
```

Comment:
Shouldn't the name still be `Optional`? That has special semantics (observing all notifications in this `NotificationCenter`).